doc/source/building/working_with_images/vagrant_setup.rst

```diff
@@ -0,0 +1,134 @@
+.. _setup_vagrant:
+
+KIWI Image Description for Vagrant
+==================================
+
+.. sidebar:: Abstract
+
+   This page provides further information for handling
+   vmx images built with KIWI and references the following
+   articles:
+
+   * :ref:`vmx`
+
+`Vagrant <http://vagrantup.com>`_ is a framework to
+implement consistent processing/testing work environments based on
+Virtualization technologies. To run a system, Vagrant needs so-called
+**boxes**. A box is a TAR archive containing a virtual disk image and
+some metadata.
+
+To build Vagrant boxes, you can use
+`veewee <https://github.com/jedi4ever/veewee>`_ which builds boxes
+based on AutoYaST, or `Packer <http://packer.io>`_, which is provided
+by Vagrant itself.
+
+Both tools are based on the official distribution media (DVDs). If such
+media does not exist (for example if the distribution is still under
+development or you want to use a collection of your own repositories), the
+KIWI way of building images might be helpful. In addition you can use the
```

Comment:
I would try to avoid parenthesis. What about this:

```
The KIWI way of building images might be helpful, if such a media does not exist.
For example, if the distribution is still under development or you want to use a
collection of your own repositories.
```